docs/project/2024/florin_mihai.prodan/index.md

```diff
@@ -46,6 +46,11 @@ I started to add the components on the breadboard, then I will also connect the
 
 ### Week 20 - 26 May
 
+This week I improved my code adding more functionalities: the money system and also some LED light show. I gave up on the PCB because I couldn't manage to solder it properly so I resumed to a easier solution: soldering the LEDs directly on my box:
+![LEDS soldering](./LEDS_soldering.jpg)
```

Comment:
Test deployment is failing because the picture name is not the same with the one you added in files. Modify this to `./LEDs_soldering.jpg`.